components/backup-stream/src/router.rs

```diff
@@ -694,28 +696,31 @@ impl TempFileKey {
         }
     }
 
-    /// path_to_log_file specifies the path of record log.
+    /// path_to_log_file specifies the path of record log for v2.
     /// ```text
-    /// v1/${date}/${hour}/${store_id}/t00000071/434098800931373064-f0251bd5-1441-499a-8f53-adc0d1057a73.log
+    /// V1: v1/${date}/${hour}/${store_id}/t00000071/434098800931373064-f0251bd5-1441-499a-8f53-adc0d1057a73.log
+    /// V2: v1/${date}/${hour}/${store_id}/434098800931373064-f0251bd5-1441-499a-8f53-adc0d1057a73.log
     /// ```
-    fn path_to_log_file(&self, store_id: u64, min_ts: u64, max_ts: u64) -> String {
+    /// For v2, we merged the small files (partition by table_id) into one file.
+    fn path_to_log_file(store_id: u64, min_ts: u64, max_ts: u64) -> String {
         format!(
-            "v1/{}/{}/{}/t{:08}/{:012}-{}.log",
+            "v1/{}/{}/{}/{:012}-{}.log",
```

Comment:
Why change v1 format? and why `{:012}` for 18 chars ts?

Reply:
Unified V1 and V2 both use the `v1/...` structure.

And after v6.3.0, only use V2.